alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/processor/notification/NotificationProcessor.java

```diff
@@ -112,6 +154,46 @@ public NotificationProcessor(ConfigurationAccessor configurationAccessor, List<P
         return List.of();
     }
 
+    private List<DistributionEvent> processNotifications(StatefulProvider statefulProvider, ProviderDistributionFilter distributionFilter, ConfigurationJobModel job, List<AlertNotificationModel> notifications) {
```

Comment:
Why is this method above the only method that invokes it?

Reply:
I'm not sure I understand your question.

Reply:
This method is invoked in the method `processNotificationsForJobs()` and `processNotificationsForJobs()` is invoked by `processNotificationsThatMatchFilter()`

Reply:
I don't remember what I was getting at here